infrastructure/lib/atat-iam-stack.ts

```diff
@@ -79,14 +78,14 @@ export class AtatIamStack extends cdk.Stack {
       description: "Grants read/write access to developer-specific actions and resources",
       statements: [
         // During a deployment, outside of CloudFormation (where the role will be used),
-        // the CDK needs to read from and write to the S3 buckets for the CDK.
+        // the CDK needs to read from and write to the S3 buckets for the
```

Comment:
Not sure if you wanted to remove more, add something else, or if it was accidentally deleted. I made a suggestion for what I thought you were trying to do. 
```suggestion
        // the CDK needs to read from and write to the S3 buckets.
```

Reply:
If I told that I did a find and replace to do `s/cdk\.//i` would it surprise you that this happened :D Great catch. And honestly, yeah, this suggestion to just end the sentence makes more sense.